src/app/main/main.component.js

```diff
@@ -53,7 +53,7 @@ function routingConfig($stateProvider, $locationProvider, $urlRouterProvider){
       template: '<your-giving></your-giving>'
     })
     .state('payment-methods', {
-      url: "/profile/payment-methods.html",
+      url: "/payment-methods.html",
```

Comment:
I thought we were prefixing the profile pages with `/profile` too. Doesn't have to be done in this PR and maybe needs more discussion...